pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -844,6 +844,26 @@ The delayed message index time step(in seconds) in per bucket snapshot segment,
                     + "it can be enabled to reduce the memory consumption caused by pendingAcks.")
     private boolean autoShrinkForConsumerPendingAcksMap = false;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            dynamic = true,
+            doc = "The minimum receive buffer size for AdaptiveRecvByteBufAllocator"
+    )
+    private int minReceiveByteBuf = 1024;
```

Comment:
Please make the config name more descriptive. Maybe like `minReceiveSizeForNettyByteBufAllocator` ?